src/TextBoxSearch.webmodeler.ts

```diff
@@ -0,0 +1,76 @@
+import { Component, ReactElement, createElement } from "react";
+import { findDOMNode } from "react-dom";
+import * as classNames from "classnames";
+import { Alert } from "./components/Alert";
+
+import { TextBoxSearch, TextBoxSearchProps } from "./components/TextBoxSearch";
+import { ContainerProps, ContainerState } from "./components/TextBoxSearchContainer";
+import { Utils, parseStyle } from "./utils/ContainerUtils";
+
+// tslint:disable-next-line class-name
+export class preview extends Component<ContainerProps, ContainerState> {
+
+    constructor(props: ContainerProps) {
+        super(props);
+
+        this.state = { listviewAvailable: true };
+    }
+
+    render() {
+        return createElement("div",
+            {
+                className: classNames("widget-text-box-search", this.props.class),
+                style: parseStyle(this.props.style)
+            },
+            this.renderAlert(),
```

Comment:
This function does validation based on the list view whose existence is not guaranteed at this point. There fore there is potential for error. Please change implementation to create the create Alert directly and provide the validation in the state at the right moment when the list view has loaded